source/_static/js/widget-sidebar.js

```diff
@@ -0,0 +1,14 @@
+(function() {
```

Comment:
Curious if you wrote all this (the custom sidebar widget js/css) or whether it came from a ready-made gist or something someone else made? If the latter, can we include a link to the source so we can keep it up to date and give credit?

Reply:
Nope, I wrote the JS bits myself.